creator-node/compose/env/base.env

```diff
@@ -13,6 +13,9 @@ redisPort=6379
 # Can be overriden.
 creatorNodeIsDebug=true
 
+# Locally we run 4 CNs so we don't want to use multiple processes for each
+expressAppConcurrency=1
```

Comment:
or... what if we ran it with higher? to make sure we find any issues locally first
also even with multiple processes on each CPU, so what right

Reply:
how about running at 2 so we test concurrency but don't overload it? could've been unrelated but when I was first testing it just froze during `A up` with the default of 4

Reply:
agreed on 2, seems concerning if things freeze up with so little load